rebar.config

```diff
@@ -1,30 +1,35 @@
 %-*-Erlang-*-
 {erl_opts, [debug_info, {parse_transform, lager_transform}]}.
-{minimum_otp_vsn, "19.0"}.
+{minimum_otp_vsn, "21.0"}.
 
 {deps, [
-    {lager,          "3.7.0"},
-    {exometer_core,  "1.5.2"}
+    {exometer_core,  "1.5.7"}
```

Comment:
Will this be replaced with Prometheus?

Reply:
no, this change is just removing "lager"